addons/binding/org.openhab.binding.avmfritz/ESH-INF/i18n/avmfritz_de.properties

```diff
@@ -0,0 +1,149 @@
+
+thing-type.config.avmfritz.FRITZ_Powerline_546E_Solo.protocol.label = Protokoll
+thing-type.config.avmfritz.FRITZ_Powerline_546E_Solo.protocol.description = Protokoll f�r den Verbindungsaufbau zum FRITZ!Powerline 546E Ger�t (http or https).
+
+thing-type.config.avmfritz.FRITZ_Powerline_546E_Solo.password.label = Passwort
+thing-type.config.avmfritz.FRITZ_Powerline_546E_Solo.password.description = Passwort zur Authentifizierung an dem FRITZ!Powerline 546E Ger�t.
+
+thing-type.config.avmfritz.FRITZ_Powerline_546E_Solo.pollingInterval.label = Abfrageintervall
+thing-type.config.avmfritz.FRITZ_Powerline_546E_Solo.pollingInterval.description = Intervall zur Abfrage des FRITZ!Powerline 546E Ger�t (in Sekunden).
+
+thing-type.config.avmfritz.FRITZ_Powerline_546E_Solo.asyncTimeout.label = Asynchroner Timeout
+thing-type.config.avmfritz.FRITZ_Powerline_546E_Solo.asyncTimeout.description = Timeout f�r asynchrone Abfragen des FRITZ!Powerline 546E Ger�t (in Millisekunden).
+
+thing-type.config.avmfritz.FRITZ_Powerline_546E_Solo.syncTimeout.label = Synchroner Timeout
+thing-type.config.avmfritz.FRITZ_Powerline_546E_Solo.syncTimeout.description = Timeout f�r synchrone Abfragen des FRITZ!Powerline 546E Ger�t (in Millisekunden).
+
+thing-type.config.avmfritz.FRITZ_Powerline_546E_Solo.ain.label = AIN
+thing-type.config.avmfritz.FRITZ_Powerline_546E_Solo.ain.description = Die AHA ID (AIN) zur Identifikation des FRITZ!Powerline 546E Ger�tes.
+
+# thing types
+thing-type.avmfritz.FRITZ_DECT_Repeater_100.description = FRITZ!DECT Repeater 100 DECT Repeater. Liefert Daten wie z.B. Temperatur.
+
+thing-type.avmfritz.FRITZ_DECT_200.description = FRITZ!DECT 200 schaltbare Steckdose. Dient zur Steuerung der integrierten Steckdose und liefert Daten wie z.B. Temperatur und Energie.
+
+thing-type.avmfritz.FRITZ_DECT_210.description = FRITZ!DECT 210 schaltbare Steckdose. Dient zur Steuerung der integrierten Steckdose und liefert Daten wie z.B. Temperatur und Energie.
+
+thing-type.avmfritz.FRITZ_DECT_300.description = FRITZ!DECT 300 Heizk�rperregler. Dient zur Steuerung von Heizk�rpern und liefert Daten wie z.B. Temperatur.
+
+thing-type.avmfritz.Comet_DECT.description = Comet DECT Heizk�rperregler. Dient zur Steuerung von Heizk�rpern und liefert Daten wie z.B. Temperatur.
+
+thing-type.avmfritz.FRITZ_Powerline_546E.description = FRITZ!Powerline 546E schaltbare Steckdose. Dient zur Steuerung der integrierten Steckdose und liefert Daten wie z.B. Temperatur.
+
+# thing types config
+thing-type.config.avmfritz.fritzdevice.ain.description = Die AHA ID (AIN) zur Identifikation des FRITZ! Ger�tes.
+
+# channel types
+channel-type.avmfritz.mode.label = Modus des Ger�tes
+channel-type.avmfritz.mode.description = Zeigt den aktuellen Modus des Ger�tes an (MANUAL/AUTOMATIC).
+channel-type.avmfritz.mode.state.option.MANUAL = Manuell
+channel-type.avmfritz.mode.state.option.AUTOMATIC = Automatisch
+
+channel-type.avmfritz.locked.label = Schalten des Ger�tes (extern)
```

Comment:
Makes sense. Especially with the background that the upcoming FRITZ!OS gives us a new channel for `devicelocked` ("Schalten am Gerät") . ;-)